content/en/docs/reference/access-authn-authz/authorization.md

```diff
@@ -134,6 +134,20 @@ The output is similar to this:
 no
 ```
 
+Also, you can use this command to determine if a Service Account can perform a certain action.
+
+To check if a Service Account named `dev-sa` in Namespace `dev` can list Pods in the Namespace `dev`,
```

Comment:
This could probably be more concise, as the impersonation docs linked above touch on usage w/ service accounts already.

Additionally, I'd consider using two different namespaces in the example.

```suggestion
Similarly, to check whether a Service Account named `dev-sa` in Namespace `dev` can list Pods in the Namespace `target`:

Reply:
@sejr +1. Thus, committing the change.